source/beamline/BioXAS/BioXASBeamline.h

```diff
@@ -165,10 +162,16 @@ class BioXASBeamline : public CLSBeamline
 
 	/// Returns the scaler.
 	virtual CLSSIS3820Scaler* scaler() const { return 0; }
+	/// Returns the I0 Keithley amplifier.
+	virtual CLSKeithley428* i0Keithley() const { return 0; }
```

Comment:
I think it's better to keep them as they are for now. Imaging will likely follow the same paradigm (i0, i1, i2) and is a subclass of BioXASBeamline, but doesn't have Keithley objects to return yet.